src/Service.Tests/SqlTests/GraphQLMutationTests/MultipleCreateMutationTests/MultipleCreateMutationTestBase.cs

```diff
@@ -0,0 +1,819 @@
+                                                    }
+                                                ]
+                                                }
+                                            ]
+                                            ) {
+                                            items {
+                                                id
+                                                title
+                                                publisher_id
+                                                publishers {
+                                                id
+                                                name
+                                                }
+                                                reviews {
+                                                items {
+                                                    book_id
+                                                    id
+                                                    content
+                                                    website_users {
+                                                    id
+                                                    username
+                                                    }
+                                                }
+                                                }
+                                                authors {
+                                                items {
+                                                    id
+                                                    name
+                                                    birthdate
+                                                }
+                                                }
+                                            }
+                                            }
+                                        }
+                                        ";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+            SqlTestHelper.PerformTestEqualJsonStrings(expectedResponse, actual.ToString());
+
+            // Validate that the records are created in the linking table
+            string actualResponseFromLinkingTable = await GetDatabaseResultAsync(linkingTableDbValidationQuery);
+            SqlTestHelper.PerformTestEqualJsonStrings(expectedResponseFromLinkingTable, actualResponseFromLinkingTable);
+        }
+
+        #endregion
+
+        #region Relationships defined through config file
+
+        /// <summary>
+        /// <code>Do: </code> Point create mutation with entities related through a 1:1 relationship. Relationship is defined through the config file.
+        /// <code>Check: </code> createUser_NonAutogenRelationshipColumn and UserProfile_NonAutogenRelationshipColumn items are successfully created in the database. UserProfile_NonAutogenRelationshipColumn item is created and linked in the database.
```

Comment:
```suggestion
        /// <code>Do: </code> Point create mutation with entities related through a 1:1 relationship. Relationship is defined through the config file.
        /// <code>Check: </code> createUser_NonAutogenRelationshipColumn and UserProfile_NonAutogenRelationshipColumn items
        /// are successfully created in the database. UserProfile_NonAutogenRelationshipColumn item is created and linked in the database.
```

> createUser_NonAutogenRelationshipColumn and UserProfile_NonAutogenRelationshipColumn items are successfully created in the database.
is self-explanatory. Would be better to detail context. 
- 1:1 relationship based on which fields? 
- which entity is created first? which field is sourced from created item to be used in next insert?
- Which entity is created second?

Reply:
Updated comments.